CharlieBackend.Core/DTO/Attachment/AttachmentDto.cs

```diff
@@ -1,14 +1,24 @@
-﻿using System.ComponentModel.DataAnnotations;
+﻿using System;
+using System.ComponentModel.DataAnnotations;
 
 namespace CharlieBackend.Core.DTO.Attachment
 {
     public class AttachmentDto
     {
+        [Required]
         public long Id { get; set; }
 
+        [Required]
+        public DateTime CreatedOn { get; set; }
```

Comment:
maybe you should add an attribute [DataType(DataType.Date)]?

Reply:
right

Reply:
but need [DataType(DataType.DateTime)] since we also need time

Reply:
you're right